src/pages/api/paperform/[id].ts

```diff
@@ -0,0 +1,29 @@
+import Adsapi from '@api';
+import { PaperFormController, PaperFormType, RawPaperFormParams } from '@controllers/paperformController';
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+export interface PaperFormRequest extends NextApiRequest {
+  body: RawPaperFormParams;
+  query: {
+    id: PaperFormType;
+  };
+}
+
+export default async (req: PaperFormRequest, res: NextApiResponse): Promise<void> => {
+  const {
+    query: { id },
+    body,
+  } = req;
+
+  const adsapi = new Adsapi({ token: req.session.userData.access_token });
+
+  try {
+    const controller = new PaperFormController(id, body, adsapi);
+    const query = await controller.getQuery();
+    res.writeHead(302, { Location: `/search?${query}` });
+  } catch (e) {
+    res.writeHead(500, { Location: '/error/server' });
```

Comment:
Where does `/error/server` go to? Is it something we can see inside `/pages` ?

Reply:
Yeah for now, I didn't make a page.  Gonna have to go back over things to try to think about error handling here